### ISSUE TITLE
Not a single opportunity with Full Node

### ISSUE THREAD

**Issue by cryptoballena** [Haehnchen/uniswap-arbitrage-flash-swap#11]
After 5 days running my own full node on AWS, I didn't get a single opportunity found. I'm monitoring 5 pairs at same time. I tried also with only 1 with no results. German zone.

I'm not sure where I should start optimizing. Any thoughts?

**Comment by mkroxm** [comment]
Can you please help me to install node or refer any step by step guide

**Comment by mindfulme** [comment]
3 weeks in mine, lots of errors. There perhaps were some opportunities that the script + node missed.  0xE25426e7938afE2D2A638f3c1ab4E2C50b01329a

**Comment by ifunanya55** [comment]
> 3 weeks in mine, lots of errors. There perhaps were some opportunities that the script + node missed. 0xE25426e7938afE2D2A638f3c1ab4E2C50b01329a

lol so much fail...is your wbnb approved already. your tx might tend to fail if your wbnb is not approved. i haven't tested it out(the script) yet

**Comment by mindfulme** [comment]
> > 3 weeks in mine, lots of errors. There perhaps were some opportunities that the script + node missed. 0xE25426e7938afE2D2A638f3c1ab4E2C50b01329a
> 
> lol so much fail...is your wbnb approved already. your tx might tend to fail if your wbnb is not approved. i haven't tested it out(the script) yet

how much BNB is really needed to execute a flashloan? I thought there is just enough of BNB (i.e. 0.01 BNB) to cover gas fees.? I  setup the script for 1 month to see what errors I get, now I continue my work on it.

**Comment by mindfulme** [comment]
> > 3 weeks in mine, lots of errors. There perhaps were some opportunities that the script + node missed. 0xE25426e7938afE2D2A638f3c1ab4E2C50b01329a
> 
> lol so much fail...is your wbnb approved already. your tx might tend to fail if your wbnb is not approved. i haven't tested it out(the script) yet

I have not even approved any WBN, I have just BNB which sits on my wallet, yet none of trx I tried succeeded

**Comment by Haehnchen** [comment]
> wbnb is not approved

there is no approval needed, as you are playing with "foreign money"; only the profit is landing inside your wallet.

> After 5 days running my own full node on AWS, I didn't get a single opportunity found. I'm monitoring 5 pairs at same time

- use DEXs which are not crowded
- wait for market volatility

**Comment by joacorob** [comment]
You must to monitoring >2k pairs @cryptoballena

**Comment by mindfulme** [comment]
yeah, lots of opportunities every block. Just make sure you catch them, cuz our bot takes most of them :)

**Comment by rguitiayoub** [comment]
> yeah, lots of opportunities every block. Just make sure you catch them, cuz our bot takes most of them :)

just funny how you are on every repo trying hard to sell your bots

**Comment by mindfulme** [comment]
not selling anymore, just consulting or whatever. Just trying to study the competition before it catches me
